google/cloud/bigquery/job/base.py

```diff
@@ -19,11 +19,12 @@
 import http
 import threading
 import typing
-from typing import ClassVar, Dict, Optional, Sequence
+from typing import Any, ClassVar, Dict, Optional, Sequence, Union
 
 from google.api_core import retry as retries
 from google.api_core import exceptions
 import google.api_core.future.polling
+from google.api_core.future.polling import PollingFuture
```

Comment:
Import modules not classes. https://google.github.io/styleguide/pyguide.html#s2.2-imports

Reply:
Thanks, cleaned up the imports.